slatedb/src/config.rs

```diff
@@ -864,6 +864,9 @@ pub struct SizeTieredCompactionSchedulerOptions {
     /// be included in a given compaction. A sorted run S will be added to a compaction C if S's
     /// size is less than this value times the min size of the runs currently included in C.
     pub include_size_threshold: f32,
+
+    /// The maximum number of concurrent compactions to execute at once.
+    pub max_concurrent_compactions: usize,
```

Comment:
Rather than this, WDYT about updating:

```rust
pub trait CompactionSchedulerSupplier: Send + Sync {
    fn compaction_scheduler(&self) -> Box<dyn CompactionScheduler + Send + Sync>;
}
```

To pass in the compactor options (which already have max_concurrent_compactions defined)?